tests/rest/client/test_shadow_banned.py

```diff
@@ -97,7 +97,12 @@ def test_invite_3pid(self) -> None:
         channel = self.make_request(
             "POST",
             "/rooms/%s/invite" % (room_id,),
-            {"id_server": "test", "medium": "email", "address": "test@test.test"},
+            {
+                "id_server": "test",
+                "medium": "email",
+                "address": "test@test.test",
+                "id_access_token": self.banned_access_token,
```

Comment:
banned_access_token is not an appropriate value here: it is an access token for the homeserver, not the id server.

it looks like it doesn't matter what the actual value is, so you may as well just pass in a dummy string.